src/Core/src/Animations/Ticker.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Timers;
+
+namespace Microsoft.Maui.Animations
+{
+	public class Ticker : ITicker
```

Comment:
I think it is useful and will be used beyond animations, move this to the Microsoft.Maui namespace?

Reply:
I put this in an animation namespace since most the controls animation stuff is intact. Namespace collisions will be a problem.